packages/@o3r/rules-engine/package.json

```diff
@@ -160,7 +160,7 @@
     "type-fest": "^4.10.2",
     "typescript": "~5.5.4",
     "typescript-eslint": "~8.17.0",
-    "typescript-json-schema": "~0.65.0",
+    "typescript-json-schema": "^0.65.0",
```

Comment:
this is a devDep. Do you want to update it in this PR?

Reply:
This is done by the harmonize because, as explained above, `~0.65.0` == `^0.65.0`.
I open the issue #2575 to keep trace of it.